trackers/core/bytetrack/kalman_box_tracker.py

```diff
@@ -0,0 +1,173 @@
+from typing import Optional, Union
+
+import numpy as np
+
+
+class ByteTrackKalmanBoxTracker:
```

Comment:
This class seems an exact copy of `DeepSORTKalmanBoxTracker`. I think it would be better if we decoupled the `SORTKalmanBoxTracker` and`DeepSORTKalmanBoxTracker` into a separate module called `kalman_box_tracker` to make sure that we don't have tightly coupled class dependencies between the individual trackers while not repeating redundant code.